xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/DiagramMacros.xml

```diff
@@ -147,6 +146,18 @@
     )))
   #end
 #end
+#macro(checkDiagramAppInstall)
+  #if (!$services.diagram)
+    {{warning}}
+      #set ($queryString = $escapetool.url({
+        'section': 'XWiki.Extensions',
+        'search': 'Diagram'
+      }))
+      #set ($installURL = $xwiki.getURL('XWiki.XWikiPreferences', 'admin', $queryString))
+      $services.rendering.escape($services.localization.render('confluencediagram.installRequired', ['[[', "&gt;&gt;path:$installURL||target='_blank']]"]), $services.rendering.resolveSyntax($xwiki.getCurrentContentSyntaxId()))
```

Comment:
See my previous comment. This will also escape the wiki syntax from the translation parameters (thus loosing the link), which you don't want.

Reply:
fixed, by removing duplication